brangelina.vim

```diff
@@ -138,6 +138,8 @@ augroup customCommands
   au FileType elm nmap <buffer> <localleader>r <Plug>(elm-repl)
   au FileType elm nmap <buffer> <localleader>d <Plug>(elm-show-docs)
   au FileType elm nmap <buffer> <localleader>D <Plug>(elm-browse-docs)
+  au FileType elm set tabstop=4
+  au FileType elm set shiftwidth=4
```

Comment:
Elm-format ensures this after saving, definitely, but I still find it slightly annoying that while writing code `<tab>` gives me different indents than what I get after formatting.